docs/tutorial/graph_api.rst

```diff
@@ -0,0 +1,656 @@
+    - One-shot operations
+  * - Launch overhead is significant (many small kernels)
+    - Kernels are long-running
+
+Transitioning a CT reconstruction pipeline
+------------------------------------------
+
+In this tutorial, we will modify an existing GPU-accelerated stream-based image processing pipeline that reconstructs
+computer tomography (CT) data (the classic Shepp-Logan phantom [ShLo74]_). The pipeline transforms raw X-ray
+projections into clear cross-sectional images used in medical diagnosis.
+
+.. figure:: ../data/tutorial/graph_api/ct_reconstruction_overview.png
+  :alt: Diagram showing raw projection data being transformed into a reconstructed CT slice
+  :align: center
+
+.. note::
+  The tutorial application generates a phantom volume and forward projections. This GPU-accelerated operation uses
+  multiple streams and appears in the traces. You can ignore the dataset generation — it is not relevant to this
+  tutorial.
+
+The reconstruction pipeline consists of:
+
+1. **Load** projection data into GPU memory
+2. **Preprocess** the projection through six stages:
+  a. Logarithmic transformation (convert X-ray intensities)
+  b. Pixel weighting (correct for cone-beam geometry)
+  c. Forward FFT (transform to frequency domain)
+  d. Shepp-Logan filtering (enhance edges and improve contrast)
+  e. Inverse FFT (return to spatial domain)
+  f. Normalization (account for unnormalized FFT)
+3. **Reconstruct** the 3D volume using the Feldkamp-Davis-Kress (FDK) algorithm [FeDK84]_
+
+**Why HIP graphs?** CT scanners process hundreds of projections per scan. By capturing this fixed workflow as a graph,
+you will reduce the amount of API calls required for launching the workflow on a GPU to 1 per projection, thus reducing
+launch overhead and dispatch latency to near-zero.
+
+What you will learn
+-------------------
+
+After completing this tutorial, you will be able to:
+
+* Convert a stream-based HIP application to use graphs via stream capturing
+* Create graphs manually for fine-grained control
+* Integrate graph-safe libraries like hipFFT into your graphs
+* Understand when graphs provide performance benefits
+* Apply graph concepts to your own workflows
+
+Before you begin
+----------------
+
+Required knowledge
+^^^^^^^^^^^^^^^^^^
+
+You should be comfortable writing and debugging HIP kernels, understand basic GPU memory management concepts like
+device allocation and host-to-device transfers, be familiar with HIP streams and events, and have experience using
+CMake to build C++ projects. This tutorial assumes you have written at least a few HIP programs before and understand
+concepts like grid dimensions and thread blocks.
+
+Hardware and software requirements
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Your system needs ROCm 6.2 or later with the hipFFT library installed. The tutorial works on
+all :doc:`supported AMD GPUs <rocm-install-on-linux:reference/system-requirements>`, though at least 4 GiB of GPU
```

Comment:
```suggestion
all :doc:`supported AMD GPUs <rocm-install-on-linux:reference/system-requirements>`, though at least 4 GB of GPU
```

Reply:
That's intentional, see https://en.wikipedia.org/wiki/Byte#Multiple-byte_units